pkg/acsengine/engine.go

```diff
@@ -1548,7 +1548,7 @@ func (t *TemplateGenerator) getTemplateFuncMap(cs *api.ContainerService) templat
 				case "etcdVersion":
 					val = cs.Properties.OrchestratorProfile.KubernetesConfig.EtcdVersion
 				case "etcdDiskSizeGB":
-					val = cs.Properties.OrchestratorProfile.KubernetesConfig.EtcdDiskSizeGB
+					val = string(cs.Properties.OrchestratorProfile.KubernetesConfig.EtcdDiskSizeGB)
 				default:
```

Comment:
@CecileRobertMichon, not sure if this is the right thing to do here. What do you think?

Reply:
You might want to use `strconv.Itoa(cs.Properties.OrchestratorProfile.KubernetesConfig.EtcdDiskSizeGB) ` like it's being done for `gchighthreshold` and `gclowthreshold` a few lines above for consistency

Reply:
Ah, I missed that! Thanks for pointing it out, I'll make the change